devicetypes/smartthings/zwave-range-extender.src/zwave-wange-extender.groovy

```diff
@@ -0,0 +1,56 @@
+/**
+ *	Copyright 2015 SmartThings
+ *
+ *	Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *	in compliance with the License. You may obtain a copy of the License at:
+ *
+ *		http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *	Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *	on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *	for the specific language governing permissions and limitations under the License.
+ *
+ */
+metadata {
+	definition (name: "Z-Wave Range Extender", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "oic.d.networking") {
```

Comment:
since this device doesn't do anything, I'd add `hidden: true` to the definition here to hide it in the OneApp UI.

Reply:
You'll still want to do this one, though.

It's the same thing we did with the `springs-window-fashions-remote` since it has no functionality in our system.

Reply:
:+1: